src/lib/service-bus/get-connector.js

```diff
@@ -0,0 +1,17 @@
+import { AzureServiceBus } from './AzureServiceBus.js';
+import { MessageSender } from 'ffc-messaging';
+
+/**
+ * Returns an instance of a ServiceBus, depending on the environment.
+ * In non-local environments, it returns a MessageSender configured with the contact details queue.
+ * In local environment, it returns a Redis instance connected to the 'redis' host on port 6379.
+ * @param app
+ * @returns {object} Message sender
+ */
+export function getConnecter(app) {
```

Comment:
I think this should be getConnector with the 'o'?

Reply:
Edited.